### ISSUE TITLE
Fix epydoc warnings in twisted.web

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8537]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8537|
|Type|enhancement|
|Created|2016-06-29 16:39:10Z|
|Branch|https://github.com/twisted/twisted/tree/8537-rodrigc-doc|

During generation of apidocs, there are some epydoc warnings in twisted.web.  These should be fixed.

<details><summary>Searchable metadata</summary>

```
trac-id__8537 8537
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8537_rodrigc_doc 8537-rodrigc-doc
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__None None
time__1467218350630574 1467218350630574
changetime__1467290614308530 1467290614308530
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/271

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks good. Thanks! Only a minor comment. Please check PR and merge.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 79d61a975ff1ab0b18dc572a108f497168c63dfb
```
#!CommitTicketReference repository="" revision="79d61a975ff1ab0b18dc572a108f497168c63dfb"
Merge pull request #278:  Fix epydoc warnings in twisted/web

Author: rodrigc
Reviewer: adiroiban
Fixes: #8537
```
